tests/e2e/specs/SmokeTest.spec.ts

```diff
@@ -40,7 +40,7 @@ suite(`The SmokeTest userstory`, async function (): Promise<void> {
             await projectAndFileTests.waitWorkspaceReadinessForCheCodeEditor();
         });
         test('Check a project folder has been created', async function (): Promise<void> {
-            const projectName: string = StringUtil.getProjectNameFromGitUrl(factoryUrl);
+            const projectName: string = FactoryTestConstants.TS_SELENIUM_PROJECT_NAME || StringUtil.getProjectNameFromGitUrl(factoryUrl);
```

Comment:
it looks like kind of workaround to problem with `StringUtil.getProjectNameFromGitUrl(factoryUrl)`.

It would be interesting to see an example when this method returns incorrect project name.

Reply:
Got it.
Makes sense.